### PR TITLE
add default currency in case bankstatement do not have ccy tag

### PR DESCRIPTION
**Pull request by bizzappdev**
Sometimes there is no ccy tag in the statement. in that case parser/processing is causing problem.
so adding default currency.
We can improve it by taking Default company currency. Bu I am not able to get user or company instance at parser level .

### CODE REVIEW

File: account_banking_camt/camt.py [hunk 1]
```diff
@@ -150,7 +150,12 @@ def parse_Stmt(self, cr, node):
                 identifier)
             )
 
-        statement.local_currency = self.xpath(node, './ns:Acct/ns:Ccy')[0].text
+        #Assuming If there is no Ccy tag then it belongs to defaut currency
```
Comment (pedrobaeza) [inline account_banking_camt/camt.py]: Put a space after `#`

File: account_banking_camt/camt.py [hunk 2]
```diff
@@ -150,7 +150,12 @@ def parse_Stmt(self, cr, node):
...
+        if not self.xpath(node, './ns:Acct/ns:Ccy'):
+            statement.local_currency = 'EUR'
```
Comment (pedrobaeza) [inline account_banking_camt/camt.py]: Better to use `statement.company_id.currency_id.id`

**Comment by pedrobaeza** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**